### ISSUE TITLE
De-couple the UI from the Backend implementation?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hapi-socketio-redis-chat-example#65]
What do people think about the idea of de-coupling the UI from the API so we can independently test different UI ?

**Comment by danger89** [comment]
Good idea, currently not the main focus I think. The main focus now should be mobile friendly (see Bootstrap) and server sessions/cookies (revering to the _null_ username). Maybe check-out:
https://github.com/rajaraodv/redispubsub#socketio-and-sticky-sessions
